validation/validation_test.go

```diff
@@ -88,6 +89,21 @@ func TestValidateBasic(t *testing.T) {
 	assert.Nil(t, runtimeInsideValidate(g))
 }
 
+func TestValidateSeccomp(t *testing.T) {
+	g := getDefaultGenerator()
+	syscallArgs := seccomp.SyscallOpts{
+		Action:   "errno",
+		Syscall:  "getcwd",
+		Index:    "0",
```

Comment:
what are `Index`, `Value` used for in this case?

Reply:
According to the code in [`parse_action.go`](https://github.com/opencontainers/runtime-tools/blob/master/generate/seccomp/parse_action.go#L32-L36):

```
	action, _ := parseAction(arguments[0])
	if action == config.DefaultAction && args.argsAreEmpty() {
		// default already set, no need to make changes
		return nil
	}
```

If args is empty, you can't add syscall.

Reply:
If you really want to test the Action, you should change the defautAction not equal to the specified action.
Then, empty args will not be a problem.

Reply:
updated.